extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/SecretVolumeConfig.java

```diff
@@ -1,21 +1,25 @@
 package io.quarkus.kubernetes.deployment;
 
+import io.quarkus.runtime.annotations.ConfigGroup;
 import io.quarkus.runtime.annotations.ConfigItem;
 
+@ConfigGroup
```

Comment:
I don't know why, but this annotation is present in similar classes (`ConfigMapVolumeConfig`, `PvcVolumeConfig`), but it wasn't here. I added it, but I don't know what I'm doing :-)

Reply:
It's correct, nice catch!